sub/sd.h

```diff
@@ -45,6 +45,7 @@ struct sd_functions {
 struct lavc_conv;
 struct lavc_conv *lavc_conv_create(struct mp_log *log, const char *codec_name,
                                    char *extradata, int extradata_len);
+void filter_SDH(struct sd *sd, char * format, int n_ignored, char * ass, int filter_harder);
```

Comment:
Filters should pick up filter-specific options on their own, instead of requiring passing them to it every time.